### ISSUE TITLE
FR: Download zip file with multiple attachments

### ISSUE THREAD

**Issue by knobunc** [eikek/docspell#1801]
If an item has multiple attachments, it would be excellent to be able to download them all in a single zip.

**Comment by eikek** [comment]
Yes, good idea! It is currently possible by selecting the only item in the search view and click "download all" from the menu. But ofc, this is not convenient at all for a single item.
